### DISCUSSION TITLE
Którą branch (gałąź) w repozytorium mamy rozwijać?

### DISCUSSION THREAD

**Discussion by speplinski** [irdpl/wcag#5] [Organizacja pracy]
Obecnie `pl-main` jest oznaczony jako gałąź domyślna (default branch).

Czy nie powinniśmy dodać `pl-WCAG-2.2`?

Dlaczego?
- `pl-main` powinna być traktowana jako gałąź główna, czyli będąca obowiązującym stanem (obecnie 2.1, dopóki 2.2 nie stanie się "standardem").
- `pl-wcag-2.1` powinna być gałęzią, gdzie wprowadźmy zmiany robocze zanim staną się wersją domyślną (`pl-main`).
- `pl-wcag-2.2` powinna być gałęzią, gdzie wprowadzamy zmiany dotyczące tej wersji (2.2). Gdy stanie się ona obowiązującą, wówczas powinna scalona (merge) do `pl-main`.

**Comment by zwiastunsw** [comment]
- `pl-main` - gałąź główna, przedstawiająca aktualny stan - pełna zgoda
- `pl-wcag-2.2` - pełna zgoda
- `pl-wcag-2.1` - nie wiem, jak podejść, bo: generalnie poprawiłbym tylko ewidentne błędy, a po opublikowaniu oficjalnej wersji i erraty, zarchiwizowałbym stan WCAG 2.1 na dzień publikacji poprawionej wersji, natomiast wszelkie zmiany w tłumaczeniu wprowadzałbym już w wersji, która się pojawi w WCAG 2.2. Ale to temat na dłuższą debatę, gdy rozstrzygniemy, czy i w jakim zakresie poprawiać to, co znalazło się w WCAG 2.1 i trafi do WCAG 2.2.
